src/Mono.Android/Android.Util/Log.cs

```diff
@@ -7,7 +8,7 @@ public partial class Log {
 
 		public static int Debug (string tag, string format, params object[] args)
 		{
-			return Debug (tag, string.Format (format, args));
+			return Debug (tag, string.Format (CultureInfo.InvariantCulture, format, args));
```

Comment:
Same comment as the `Wtf()` overload: *should* this be using `.InvariantCulture`?  I'm unconvinced.